apps/total-typescript/process.d.ts

```diff
@@ -0,0 +1,18 @@
+declare namespace NodeJS {
+  export interface ProcessEnv {
+    NEXTAUTH_URL: string
+    NEXTAUTH_SECRET: string
+    NEXT_PUBLIC_SITE_TITLE: string
+    NEXT_PUBLIC_URL: string
+    NEXT_PUBLIC_PARTNER_FIRST_NAME: string
+    NEXT_PUBLIC_PARTNER_LAST_NAME: string
+    NEXT_PUBLIC_PARTNER_TWITTER: string
+    NEXT_PUBLIC_SEO_KEYWORDS: string
+    NEXT_PUBLIC_PRODUCT_DESCRIPTION: string
+    NEXT_PUBLIC_CONVERTKIT_SIGNUP_FORM: string
+    NEXT_PUBLIC_CONVERTKIT_TOKEN: string
+    NEXT_PUBLIC_CONVERTKIT_SUBSCRIBER_KEY: string
+    CONVERTKIT_BASE_URL: string
+    NEXT_PUBLIC_GOOGLE_ANALYTICS: string
+  }
+}
```

Comment:
if these are all required because of IDE/compile errors, can they get added to the template?